buzzfeed-style-quiz/src/script.js

```diff
@@ -1,102 +1,126 @@
 //Big thanks to tgallimore for graciously providing the
 //groundwork for this code skeleton!
-let questions = { //This should be changed!
-  1 : {'a':1,'b':2,'c':3,'d':4},
-  2:  {'a':2,'b':3,'c':4,'d':1},
-  3:  {'a':2,'b':4,'c':1,'d':3},
-  4:  {'a':3,'b':1,'c':2,'d':4},
-  5:  {'a':1,'b':2,'c':3,'d':4},
-  6:  {'a':1,'b':2,'c':3,'d':4}
+let questions = {
+  //This should be changed!
+  'Q1':  {'a':1,'b':2,'c':3,'d':4},
+  'Q2':  {'a':2,'b':3,'c':4,'d':1},
+  'Q3':  {'a':2,'b':4,'c':1,'d':3},
+  'Q4':  {'a':3,'b':1,'c':2,'d':4},
+  'Q5':  {'a':1,'b':2,'c':3,'d':4},
+  'Q6':  {'a':1,'b':2,'c':3,'d':4}
```

Comment:
I opted to change the top level keys here to strings `1` => `Q1`

It reduces some ambiguity and thus makes describing the different levels in our nested dictionary a little easier.